data/json/items/ammo.json

```diff
@@ -2301,8 +2301,7 @@
       "id": "ammolink223",
       "copy-from": "ammolink",
       "type": "AMMO",
-      "name": "ammo belt linkage (223)",
-      "name_plural": "ammo belt linkages (223)",
+      "name": ".223 ammo belt linkage",
```

Comment:
I agree this is an improvement but is outside the scope of your PR. Consider factoring out as part of a second PR.